src/domain/geoResources.js

```diff
@@ -178,6 +179,14 @@ export class GeoResource {
 		return this._exportable;
 	}
 
+	/**
+	 * Returns a list of timestamps which for this GeoResource.
```

Comment:
Returns a list of timestamps which are for this GeoResource.
or 
Returns a list of timestamps that apply to this GeoResource.